flagsmith-jira-app/src/flagsmith.ts

```diff
@@ -166,14 +172,16 @@ export const fetchFeatures = async ({
 
 export const fetchFlags = async ({
   apiKey,
-  environmentId,
+  featureName,
+  envAPIKey,
 }: {
   apiKey: string;
-  environmentId: string | undefined;
-}): Promise<FlagModel[]> => {
+  featureName: string;
+  envAPIKey: string;
+}): Promise<FlagModel> => {
   checkApiKey(apiKey);
-  if (!environmentId) throw new ApiError("Flagsmith environment not configured", 400);
-  const path = route`/features/featurestates/?environment=${environmentId}`;
-  const data = (await flagsmithApi(apiKey, path)) as PaginatedModels<FlagModel>;
-  return await unpaginate(apiKey, data);
+  if (!featureName) throw new ApiError("Flagsmith environment not configured", 400);
+  const path = route`/flags/?feature=${featureName}`;
```

Comment:
Can you explain why we had to also change this endpoint? I'd rather not use the SDK endpoints here if we can avoid it. [This](https://github.com/Flagsmith/flagsmith/issues/3301) issue might also be relevant here.

Reply:
It was an initial idea to individually obtain the data of a specific feature, now it was changed to another endpoint: `/environments/${envAPIKey}/featurestates/?feature_name=${featureName}`